Bio/SeqFeature.py

```diff
@@ -335,6 +335,72 @@ def extract(self, parent_sequence):
                              "sequence file for a valid location.")
         return self.location.extract(parent_sequence)
 
+    def translate(self, parent_sequence, table="Standard", stop_symbol="*",
+            codon_start=1, to_stop=False, cds=False, gap=None):
+        """Get a translation of the feature's sequence.
```

Comment:
I think the docstring should say this is intended for CDS type features (only), although I'm not sure if we should enforce that in the code - there may be value in trying to translate other feature types (e.g. misc. features)?